code-studio/src/js/components/progress/course_progress_row.jsx

```diff
@@ -80,15 +96,13 @@ const CourseProgressRow = React.createClass({
         {ribbon}
         <div style={styles.stageName}>
           {stage.title}
-          <div className='stage-lesson-plan-link' style={{display: 'none'}}>
-            <a target='_blank' href={stage.lesson_plan_html_url}>
-              {dashboard.i18n.t('view_lesson_plan')}
-            </a>
-          </div>
+          {lessonPlanLink}
```

Comment:
The way you have this is fine. That said, in my own React code I've been trying to keep as much as the JSX in one place as I can, as I think it makes it easier to visualize what the component is actually rendering. Some of this is just personal preference, and you're welcome to do whatever feels better to you. In this case, that would look something like this:
```jsx
        <div style={styles.stageName}>
          {stage.title}
          {this.props.showLessonPlanLinks && stage.lesson_plan_html_url
            <a
                target='_blank'
                href={stage.lesson_plan_html_url}
                style={styles.lessonPlanLink}
            >
              {dashboard.i18n.t('view_lesson_plan')}
            </a>
          }
        </div>
```
(We also have a preference for splitting out one prop per line when we're passing in multiple props).